pkg/numaresourcesscheduler/objectstate/sched/sched.go

```diff
@@ -180,6 +181,45 @@ func SchedulerNameFromObject(obj client.Object) (string, bool) {
 	return "", false
 }
 
+func UpdateSchedulerName(cm *corev1.ConfigMap, name string) error {
+	if name == "" {
+		return nil
```

Comment:
let's error out here. Empty scheduler name is almost always a mistake unless we have a crystal clear use case for it (nothing in sight)